anda/tools/cbmem/cbmem.spec

```diff
@@ -0,0 +1,30 @@
+%define debug_package %nil
+
+Name:           cbmem
+Version:        24.12
+Release:        1%?dist
+Summary:        Prints out coreboot mem table information
+URL:            https://review.coreboot.org
+License:        GPLv2
+BuildRequires:  gcc g++ gcc-gnat make cmake ncurses-devel iasl git anda-srpm-macros
+Packager:       Owen Zimmerman <owen@fyralabs.com>
+
+%description
+Prints out coreboot mem table information in JSON by default, and also implements the basic cbmem -list and -console commands.
+
+%prep
+%git_clone %url/coreboot %version
+
+%build
+cd util/cbmem
+%make_build -std=c17
```

Comment:
do not include `-std=c17` here, please instead do a `%global build_cflags %__build_flags_lang_c %?_distro_extra_cflags -std=c17` at the top